.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Java with Gradle CI
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
Every PR should be checked.  When you update something in the branch, the checks must be run to make sure main is working.

Reply:
Done!